manager/controllers/mockup/datacatalog.go

```diff
@@ -52,6 +52,13 @@ func (d *DataCatalogDummy) CreateAsset(in *datacatalog.CreateAssetRequest, creds
 	return &datacatalog.CreateAssetResponse{AssetID: "testAssetID"}, nil
 }
 
+func (m *DataCatalogDummy) DeleteAsset(in *datacatalog.DeleteAssetRequest, creds string) (*datacatalog.DeleteAssetResponse, error) {
+	// TODO
+	// will be provide a proper implementation once the implementation of DeleteAssetInfo in katalog-connector
+	// is completed in a future PR. Till then a dummy implementation is provided.
+	return &datacatalog.DeleteAssetResponse{Status: "DeleteAssetInfo not implemented in DataCatalogDummy"}, nil
+}
```

Comment:
This is a mockup - not related to katalog-connector

Reply:
BTW, where is the function definition in the katalog-connector?

Reply:
In the next PR, we will provide the implementation for katalog-connector.